sdk/test/opentelemetry/sdk/trace/export/batch_span_processor_test.rb

```diff
@@ -222,7 +222,7 @@ def to_span_data
       _(test_exporter.failed_batches.size).must_equal(0)
       _(test_exporter.batches.size).must_equal(0)
 
-      _(bsp.instance_variable_get(:@spans).size).must_equal(1)
+      _(bsp.instance_variable_get(:@spans).size).must_equal(0)
```

Comment:
Since we are now properly dropping spans during `shutdown` we change the test expectation